persimmon/website/views.py

```diff
@@ -133,3 +144,49 @@ def get_my_accounts(request):
         'balance': account.balance,
         'approval_status': account.approval_status,
     } for account in accounts]
+
+
+@api_function
+def approve_credit_debit_funds(request, account_number: int, approved: bool, credittype0debittype1: int):
```

Comment:
this is kind of a strange way to structure the API - what if we want to approve only some of the pending transactions from an account and decline others? `account_number` should be replaced with `transaction_id`. (we should add an `id` field to `BankStatements`)

Reply:
make sense!